src/custom_sql_types.rs

```diff
@@ -173,3 +173,37 @@ pub enum EntityTypeEnum {
     Template,
     Test,
 }
+
+/// Maps to the custom type `machine_type_enum` in the DB
+///
+/// Represents the enum used in the DB for representing google cloud build's machine_type option
+/// for building on a different machine than the default
+///
+/// Google's Cloud Build docs explain the machine-type argument here:
+/// https://cloud.google.com/sdk/gcloud/reference/builds/submit#--machine-type
+/// Notably, it mentions two other machine types: e2-highcpu-8 and e2-highcpu-32, but, when I
+/// tested building with those, I got an error message saying the machine-type was
+/// unrecognized so ¯\_(ツ)_/¯
+#[derive(Debug, PartialEq, DbEnum, Serialize, Deserialize, Clone, Copy)]
+#[DieselType = "Machine_type_enum"]
+pub enum MachineTypeEnum {
+    #[serde(rename = "n1-highcpu-8")]
```

Comment:
So does this store the value in the DB with `-` instead of as an identifier?  Why is that desireable/necessary?

Reply:
The version with dashes is the format google cloud build actually uses, so I wanted what's in the database to match that, so using the value from the database is easier and users see the actual value that is submitted.